### PR TITLE
fix(azure): Update Azure locations

### PR DESCRIPTION
**Pull request by NeverovDS**

### CODE REVIEW

File: internal/providers/terraform/azure/app_service_custom_hostname_binding.go [hunk 1]
```diff
@@ -21,12 +21,7 @@ func GetAzureRMAppServiceCustomHostnameBindingRegistryItem() *schema.RegistryIte
 
 func NewAzureRMAppServiceCustomHostnameBinding(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
 	var sslType, sslState string
-
-	region := "Global"
-	group := d.References("resource_group_name")
-	if len(group) > 0 {
-		region = group[0].Get("location").String()
-	}
```
Comment (alikhajeh1) [inline internal/providers/terraform/azure/app_service_custom_hostname_binding.go:29]: plz revert this change, it was fine since we want this to default to Global.

**Comment by NeverovDS** [comment]
@alikhajeh1 ready

**Review by alikhajeh1 — COMMENTED** [review]
LGTM - one tiny issue
